schemas/core/objects/navigationItem.ts

```diff
@@ -39,19 +40,9 @@ export default {
          }
     },
     {
-      title: 'Check Box',
-      name: 'checkBox',
-       type: 'string',
-      options: {
-          list: [
-              { title: 'Internal',value: 'internal' },
-              { title: 'External',value: 'external' },
-              {title:"Dialog",value:"dialog"}
-          ],
-          layout:'radio',
-          isHighlighted:true
-      }
-  },
+      ...linkTypeField({ name: "urlType", title: "Url Type"}),
```

Comment:
Please attach a screen shot of the sanity component and data

Reply:
![image](https://user-images.githubusercontent.com/100187348/224048613-a01f56ab-3103-4e57-ba8a-3c6239210dc0.png)

primaryAction": {
        "_type": "navigationItem",
        "title": "MORE",
        "url": "/details-rooms",
        "variant": "light-outlined",
        "urlType": "internal",
        "image": {
          "_type": "image",
          "asset": {
            "_type": "reference",
            "_ref": "image-b588a2e9b875a417e7e86c74a82c1fbc0272279b-116x70-png"
          }
        }